components/post_view/post_list_virtualized/post_list_virtualized.jsx

```diff
@@ -443,6 +577,88 @@ class PostList extends React.PureComponent {
         this.listRef.current.scrollToItem(0, 'end');
     }
 
+    scrollToUnread = () => {
+        this.listRef.current.scrollToItem(PostList.getNewMessageIndex(this.state.postListIds), 'start', OFFSET_TO_SHOW_TOAST);
+        this.hideNewMessagesToast();
+    }
+
+    newMessagesToastText = (count, since) => {
+        let msgSince = '';
+        if (typeof since !== 'undefined') {
+            msgSince = (
+                <FormattedMessage
+                    id='postlist.toast.since'
+                    defaultMessage=' since {date} at {time}'
```

Comment:
I don't think we can combine these two sentences in all languages. Instead, we could have two strings, one for desktop and one for mobile and switch between them as necessary.

Reply:
Aah sorry, forgot about this. WIll update this